student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(name, parent_permission, age)
```

Comment:
- [ ] Kindly arrange your `#super` method arguments based on the of the inherited Person class, to avoid mixing up values.

- [ ] The `parent_permission` parameters is not correctly assigned also in the `#super` argument

  ```
  super(age, name, parent_permission: parent_permission)
  ```